golang-github-prometheus-node_exporter.spec

```diff
@@ -37,7 +37,6 @@ BuildRequires: prometheus-promu
 %package -n %{project}-node-exporter
 Summary:        %{summary}
 Provides:       prometheus-node-exporter = %{version}-%{release}
-Provides:       prometheus-node_exporter = %{version}-%{release}
 Obsoletes:      prometheus-node_exporter < %{version}-%{release}
```

Comment:
I did this because the package was renamed to use a dash instead of an underscore.  So a search for example to find what provides node_exporter will still work.

Reply:
Oops I missed the '-'/'_' difference... Will retract it then.